File: cpp/Easy/1470_shuffle_the_array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int>sh;
        for(int i = 0; i+n<nums.size(); i++){
            sh.push_back(nums[i]);
            sh.push_back(nums[i+n]);
        }
        return sh;
    }
};

File: python/easy/1470_shuffle_the_array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        ans = []
        length = len(nums)/2
        i = 0
        j = int(length)
        while i < length:
            ans.append(nums[i])
            ans.append(nums[i+j])
            i+=1
        return ans

